### PR TITLE
#12 Add current place

### LINKED ISSUE #12: Add current place component

### PR DESCRIPTION
**Pull request by hananina**
closes https://github.com/chingu-voyages/v25-geckos-team-04/issues/12

Added `place` property, which can be the place title

This is used like this:
```
<CurrentPlace place='hoge' />
```

OLD
スクリーンショット 2020-11-14 23 43 45

Updated✨
スクリーンショット 2020-11-15 15 58 22

### CODE REVIEW

File: src/components/CurrentPlace/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+interface ICurrentPlace {
+  place: string
+}
+
+export const CurrentPlace: FC<ICurrentPlace> = ({ place }) => {
+  return (
+    <>
+      <div className={styles.place}>
+        <PlaceIcon fontSize="default" />
```
Comment (inovux) [inline src/components/CurrentPlace/index.tsx]: Looking at the screenshot I feel that the Icon is too small compared to the design? Maybe you can use inherit for `fontSize` and wrap it with another element that you give a className and write a style with font size for that className. Let me know what you think :)

**Review by inovux — CHANGES_REQUESTED** [review]

File: src/components/CurrentPlace/index.tsx (hunk 1 above)
Comment (hananina) [reply src/components/CurrentPlace/index.tsx]: Thanks🙏 I will check the design later😊

Comment (hananina) [reply src/components/CurrentPlace/index.tsx]: @inovux It was smaller compare to the figma design... I updated the icon size, 15px -> 20px 👍 Thanks! 
スクリーンショット 2020-11-15 15 58 22

**Review by mrpeterhanley — APPROVED** [review]
Ok for me

**Review by inovux — APPROVED** [review]
Looks good to me :)

**Pull request merged**